sso-kit-starter-hilla/src/test/java/dev/hilla/sso/starter/BackChannelLogoutSubscriptionTest.java

```diff
@@ -12,38 +12,40 @@
 import java.util.ArrayList;
 import java.util.function.Consumer;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.verify;
+import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.test.util.ReflectionTestUtils;
 
 import com.vaadin.sso.core.UserLogoutEvent;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
 /**
  * Test suite for {@link BackChannelLogoutSubscription}.
  */
 @ExtendWith(MockitoExtension.class)
 public class BackChannelLogoutSubscriptionTest {
+
+    @Mock
+    private Consumer<Object> consumer;
+
     /**
      * Test BackChannelLogoutSubscription constructor
      */
     @Test
-    public void testBackChannelLogoutSubscriptionConstructor() {
+    public void backChannelLogoutSubscriptionConstructor_createsFlux() {
```

Comment:
This test can't ever fail: it tests that a final field is initialized by the constructor; also, it is an implementation detail.

Reply:
Removed this test.